src/main/java/hudson/plugins/xvnc/DisplayAllocator.java

```diff
@@ -18,6 +14,9 @@
  * @author Kohsuke Kawaguchi
  */
 final class DisplayAllocator {
+
+    transient Saveable owner;
```

Comment:
It is set not only after calling the `DisplayAllocator` constructor, but also from `DescriptorImpl.load`. Since the field (and whole class) is package-private, encapsulation does not seem important to me.